uber/site_sections/preregistration.py

```diff
@@ -191,7 +191,8 @@ def form(self, session, message='', edit_id=None, **params):
                     if group.badges:
                         Tracking.track(track_type, group)
 
```

Comment:
Oh, whoops! That's actually in there from the at-door and pre-reg merge, which was also in ac-staging. I'll take it out and then merge this.